internal/utils/tag.go

```diff
@@ -0,0 +1,51 @@
+package utils
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/fatih/structtag"
+)
+
+func RemoveTagsOmitempty(s interface{}, key string) (interface{}, error) {
```

Comment:
This method looks not very obvious, it would be good to have unit test and documentation string for it